src/psygnal/_group.py

```diff
@@ -403,6 +411,12 @@ def __repr__(self) -> str:
         name = self.__class__.__name__
         return f"<SignalGroup {name!r} with {len(self)} signals>"
 
+    def get_signal_by_alias(self, name: str) -> SignalInstance | None:
+        sig_name = self._psygnal_aliases.get(name, name)
+        if sig_name is None or sig_name not in self:
+            return None
+        return self[sig_name]
```

Comment:
hey @getzze, I think this is probably my only question (in an extremely nice PR).  I can see why you did it this way as opposed to modifing `__getitem__`  It allows someone to query both the alias name and the original name.  Do you think that's important? (I assume so, since you've been giving it a lot of thought and opted for this).

Reply:
hey, I think it can be removed, I mean inlined in `evented_setattr`, as it is only used there. And `SignalGroup` has one less method.

Reply:
i do think that end-users should also be able to search by alias.  but i guess the question is should they no longer be able to search by the original name?  i.e. if I remove the method on SignalGroup, then end-users can no longer access the original names for an aliased signal, correct?  are we ok with that?

Reply:
I made a function instead of a method, with a new name `get_signal_from_field`.